test/Feature/CleanRegex/_prepared/inject/PatternTest.php

```diff
@@ -271,7 +271,7 @@ public function shouldThrowForAlterationFigure()
         $this->expectException(InvalidArgumentException::class);
         $this->expectExceptionMessage('Invalid inject figure type. Expected string, but array (2) given');
         // when
-        Pattern::inject('', [['foo', 'bar']]);
+        Pattern::inject('', [['foo', 'bar']]); // @phpstan-ignore-line Deliberate testing of invalid type
```

Comment:
@veelkoov I don't like this assumption as well.

In my opinion it's perfectly fine to try these values in tests. PHP types aren't perfect, and as such allow improper values. We need to handle that in the library, and if we are to handle it, we must test it, and to test it we must pass invalid values  - so it's an unavoidable consequence (unless PHP adds generics).

What do you say we skip phpstan in `test/` for now, and we get back to it in the next PR?

Reply:
This is expected, it's just an additional "expect wrong type" for PHPStan, like expectException() for PHPUnit. It's just used on different occasion.

RE: `test/` - in another comment.